packages/example/pages/_app.tsx

```diff
@@ -24,61 +24,62 @@ function MyApp({ Component, pageProps, router }: AppProps): React.ReactElement {
-          // hideFromDefaults: [
-          //   SupportedProviders.MetaMask,
-          //   SupportedProviders.Valora,
-          //   SupportedProviders.CeloExtensionWallet
-          // ],
-          // hideFromDefaults: true,
-          sort: (a: Provider, b: Provider) =>
-            a.name === 'Dummy Wallet' ? -1 : a.name.length - b.name.length,
-          additionalWCWallets: [
-            {
-              id: 'dummy-wallet',
-              name: 'Dummy Wallet',
-              description: 'Lorem ipsum',
-              homepage: 'string',
-              chains: ['eip:4220'],
-              versions: ['1', '2'],
-              logos: {
-                sm: 'https://via.placeholder.com/40/000000/FFFFFF',
-                md: 'https://via.placeholder.com/80/000000/FFFFFF',
-                lg: 'https://via.placeholder.com/160/000000/FFFFFF',
-              },
-              app: {
-                browser: 'string',
-                ios: 'string',
-                android: 'string',
-                mac: 'string',
-                windows: 'string',
-                linux: 'string',
-              },
-              mobile: {
-                native: 'string',
-                universal: 'string',
-              },
-              desktop: {
-                native: 'string',
-                universal: 'string',
-              },
-              metadata: {
-                shortName: 'string',
-                colors: {
-                  primary: 'string',
-                  secondary: 'string',
-                },
-              },
-              responsive: {
-                mobileFriendly: false,
-                browserFriendly: true,
-                mobileOnly: false,
-                browserOnly: true,
-              },
-            },
-          ],
-        },
+        providersOptions: { searchable: true },
+        // title: <span>Connect your DummyWallet</span>,
+        // providersOptions: {
+        //   // hideFromDefaults: [
+        //   //   SupportedProviders.MetaMask,
+        //   //   SupportedProviders.Valora,
+        //   //   SupportedProviders.CeloExtensionWallet
+        //   // ],
+        //   // hideFromDefaults: true,
+        //   sort: (a: Provider, b: Provider) =>
+        //     a.name === 'Dummy Wallet' ? -1 : a.name.length - b.name.length,
+        //   additionalWCWallets: [
+        //     {
+        //       id: 'dummy-wallet',
+        //       name: 'Dummy Wallet',
+        //       description: 'Lorem ipsum',
+        //       homepage: 'string',
+        //       chains: ['eip:4220'],
+        //       versions: ['1', '2'],
+        //       logos: {
+        //         sm: 'https://via.placeholder.com/40/000000/FFFFFF',
+        //         md: 'https://via.placeholder.com/80/000000/FFFFFF',
+        //         lg: 'https://via.placeholder.com/160/000000/FFFFFF',
+        //       },
+        //       app: {
+        //         browser: 'string',
+        //         ios: 'string',
```

Comment:
delete?

Reply:
I need to uncomment it or delete, it's part of the example app that was showing customization.